### DISCUSSION TITLE
Stopped working in colab environment

### DISCUSSION THREAD

**Discussion by dgoryeo** [SYSTRAN/faster-whisper#633] [Q&A]
Running in colab, I have just started getting this warning and then nothing happens: 
```

/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:72: UserWarning: 
The secret `HF_TOKEN` does not exist in your Colab secrets.
To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.
You will be able to reuse this secret in all of your notebooks.
Please note that authentication is recommended but still optional to access public models or datasets.
  warnings.warn(
```

This started from today --I think colab has just made an update.  Has anyone else come across this? Any workaround?

**Comment by kojihashimoto-kobe** [comment]
I solved the problem by following these steps
1) Run the cell once.
2) Select 'tools' (in the menu bar above) --> 'Command palette'.
3) Select 'use fallback runtime version'.
4) Run the cell again.

**Reply by dgoryeo** [reply]
Thanks for the workaround! 
However colab fallback is temporary --though google colab engineers say they try their best to extend it as much as they can. 

Bay any chance have you found out what causes the actual problem?

**Comment by dgoryeo** [comment]
Here is a quick fix pointed out in another thread: [Stuck on model_load](https://github.com/m-bain/whisperX/issues/656#issuecomment-1877955404)

**Comment by shkstar** [comment]
I get error msg
m4a
/usr/local/lib/python3.10/dist-packages/pyannote/audio/utils/reproducibility.py:74: ReproducibilityWarning: TensorFloat-32 (TF32) has been disabled as it might lead to reproducibility issues and lower accuracy.
It can be re-enabled by calling
   >>> import torch
   >>> torch.backends.cuda.matmul.allow_tf32 = True
   >>> torch.backends.cudnn.allow_tf32 = True
See https://github.com/pyannote/pyannote-audio/issues/1370 for more details.

  warnings.warn(

And all the colab example for WhisperX and Faster whisper is not working since yesterday, anybody know why?
